test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -84,27 +91,31 @@ var (
 	}
 )
 
-// testCase defines a set of test cases that intend to test the performance of
+// testCase defines a set of test cases that intends to test the performance of
 // similar workloads of varying sizes with shared overall settings such as
 // feature gates and metrics collected.
 type testCase struct {
 	// Name of the testCase.
 	Name string
-	// Feature gates to set before running the test. Optional.
+	// Feature gates to set before running the test.
+	// Optional
 	FeatureGates map[featuregate.Feature]bool
-	// List of metrics to collect. Optional, defaults to
+	// List of metrics to collect. Defaults to
 	// defaultMetricsCollectorConfig if unspecified.
+	// Optional
 	MetricsCollectorConfig *metricsCollectorConfig
 	// Template for sequence of ops that each workload must follow. Each op will
 	// be executed serially one after another. Each element of the list must be
 	// createNodesOp, createPodsOp, or barrierOp.
 	WorkloadTemplate []op
 	// List of workloads to run under this testCase.
 	Workloads []*workload
-	// SchedulerConfigFile is the path of scheduler configuration
-	SchedulerConfigFile string
-	// Default path to spec file describing the pods to create. Optional.
+	// SchedulerConfigPath is the path of scheduler configuration
+	// Optional
+	SchedulerConfigPath *string
```

Comment:
Were there no YAML files using this field?

Reply:
Yes, search the `schedulerConfigFile` with no result return.